src/components/Calculator.jsx

```diff
@@ -1,69 +1,97 @@
           5
-        </div>
-        <div className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           6
-        </div>
-        <div className="bg-orange-500 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-orange-500 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           -
-        </div>
-        <div className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           1
-        </div>
-        <div className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           2
-        </div>
-        <div className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           3
-        </div>
-        <div className="bg-orange-500 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-orange-500 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           +
-        </div>
-        <div className="bg-gray-300 h-20 col-span-2 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-gray-300 h-20 col-span-2 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           0
-        </div>
-        <div className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-gray-300 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           .
-        </div>
-        <div className="bg-orange-500 h-20 flex items-center justify-center border-solid border-2 border-white">
+        </button>
+        <button type="button" className="bg-orange-500 h-20 flex items-center justify-center border-solid border-2 border-white" onClick={clickedButton}>
           =
-        </div>
+        </button>
```

Comment:
- [`OPTIONAL`] Great job on implementing the calculator as specified in the project requirements! 👏 However, I'd like to suggest a cleaner and more readable approach. Consider creating an array for your calculator operations and utilizing the `map()` function to iterate through the array and dynamically create each button accordingly. This will help you avoid repeating similar code for each operation button, making your code more concise and maintainable. It's a good practice to keep your code `DRY` (Don't Repeat Yourself) to enhance readability and reduce the risk of introducing bugs in the future. Keep up the good work!